consul/metadata.csv

```diff
@@ -7,3 +7,33 @@ consul.catalog.services_critical,gauge,,service,,Total critical services on node
 consul.catalog.services_passing,gauge,,service,,Total passing services on nodes,1,consul,svc pass
 consul.catalog.services_up,gauge,,service,,Total services registered on nodes,0,consul,svc up
 consul.catalog.services_warning,gauge,,service,,Total warning services on nodes,-1,consul,svc warn
+consul.net.node.latency.min,gauge,,millisecond,,minimum latency from this node to all others,-1,consul,min latency
+consul.net.node.latency.p25,gauge,,millisecond,,p25 latency from this node to all others,-1,consul,p25 latency
+consul.net.node.latency.median,gauge,,millisecond,,median latency from this node to all others,-1,consul,median latency
+consul.net.node.latency.p75,gauge,,millisecond,,p75 latency from this node to all others,-1,consul,p75 latency
+consul.net.node.latency.p90,gauge,,millisecond,,p90 latency from this node to all others,-1,consul,p90 latency
+consul.net.node.latency.p95,gauge,,millisecond,,p95 latency from this node to all others,-1,consul,p95 latency
+consul.net.node.latency.p99,gauge,,millisecond,,p99 latency from this node to all others,-1,consul,p99 latency
+consul.net.node.latency.max,gauge,,millisecond,,maximum latency from this node to all others,-1,consul,max latency
+consul.runtime.num_goroutines,gauge,10,,,Number of running goroutines,0,consul,num goroutines
```

Comment:
Do we know for sure the interval is 10? (ie. the metrics are refreshed on the consul side every 10 seconds)

Reply:
So the piece of information that was missing: https://www.consul.io/docs/agent/telemetry.html

The source of these metrics is not our check, but consul itself sending the telemetry to a statsd server, so indeed these 10 seconds interval are correct as per the metada wiki documentation.